.github/workflows/design-system-deploy.yml

```diff
@@ -32,10 +32,12 @@ jobs:
           yarn workspace @talend/assets-api run build:lib
           yarn workspace @talend/icons run build:lib
           yarn workspace @talend/design-tokens run build:lib
-          yarn workspace @talend/design-tokens run build-storybook
+          yarn workspace @talend/design-system run build:lib
+          yarn workspace @talend/storybook-docs run build:lib
+          yarn workspace @talend/design-tokens run test:demo
 
       - name: Build Storybook of the design system
-        run: yarn workspace @talend/design-system run build-storybook
+        run: yarn workspace @talend/design-system run test:demo
```

Comment:
This holds the eventual `build-storybook` I suppose?

Reply:
Yep, from the package scripts:
```
"test:demo": "npm run build-storybook",
```

Reply:
I prefer use same script every where in the CI.
we have build:lib, test, test:demo, pre-release